eng/deploy/azure/README.md

```diff
@@ -13,79 +13,58 @@ CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli).
 - [Create an ACI Docker
   context](https://docs.docker.com/cloud/aci-integration/#create-an-aci-context)
 
-- Switch to the ACI Context
+- Browse to `../eng/deploy/azure`
 
-```Shell
-docker context use myacicontext
-```
+- Create a .env file. Update URLs to match your correct Azure region.
 
-- Login to Azure Container Registry
+- Execute the following script:
 
 ```Shell
-az acr login --name edfimeadowlark
-```
-
-- Browse to the deploys folder `cd ../eng/deploy/azure`
 
-- Create a .env file. Update URLs to match your correct Azure region.
+# Switch to the ACI Context
+docker context use myacicontext
 
-- Deploy containers
+# Login to Azure Container Registry
+az acr login --name edfimeadowlark
 
-```Shell
 docker compose -p meadowlark --file ./azure-docker-compose.yml up -d
-```
-
-- Initialize mongodb replica set
 
-```Shell
+# Initialize replica set
 az container exec --resource-group {resource group name} -n meadowlark
 --container-name ml-mongo1 --exec-command 'mongo --eval rs.initiate()'
```

Comment:
Won't this fail without a line continuation character?
```suggestion
az container exec --resource-group {resource group name} -n meadowlark `
    --container-name ml-mongo1 --exec-command 'mongo --eval rs.initiate()'
```

Annoying thing is that I entered the PowerShell line continuation, which won't work for Bash. And vice versa. I'm OK with that though.